packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -295,28 +297,39 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
       // const iconButtonStyles = this._classNames.subComponentStyles
       // ? (this._classNames.subComponentStyles.iconButton as IStyleFunctionOrObject<IButtonStyleProps, IButtonStyles>)
       // : undefined;
+      const headerTextId = this.props.headerText && this.state.id + '-headerText';
+      const header = this._onRenderHeader(this.props, this._onRenderHeader, headerTextId);
```

Comment:
is this necessary? seems we could have a single function that determines headerTextID and renders the control, and share that between both render locations.

Reply:
added `_header` as a private member whose value is set within the `render` function but is conditionally rendered either in the render function in its original location or within `_onRenderNavigationContent`